packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -49,6 +58,7 @@ export class DropdownToggleCheckbox extends React.Component<DropdownToggleCheckb
           disabled={isDisabled}
           defaultChecked={isChecked || checked}
         />
+        {text}
```

Comment:
You can just put the whole statement here:

```
children && (
<span
  className={css(styles.dropdownToggleText, className)}
  aria-hidden="true"
  id={`${props.id}-text`}
>
  {children}
</span>
)
```

Reply:
Yeah, I did that and the TS lint complained about `jsx-no-multiline-js` https://github.com/palantir/tslint-react

Reply:
I think you need the `(` and `)` to comply with that linting rule in the future. I'm fine with it either way.